index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>10-Gano Courses</title>
+</head>
+<body>
+  <nav class="black-navbar">
+    <ul class="social-icons-container">
+      <li><img src="assets/Rectangle.png" alt="fb-icon"></li>
+      <li><img src="assets/twitter.png" alt="twitter-icon"></li>
+      <li><a>English</a></li>
+      <li><a>My Page</a></li>
+      <li><a>Logout</a></li>
+    </ul>
+  </nav>
+  <nav class="navbar" id="navbar">
+    <div class="logo-container">
+      <a href="index.html">
+        <img src="assets/10-gano-high-resolution-logo-black-on-transparent-background.png" alt="cc-logo" />
+      </a>
+    </div>
+    <div class="bars" id="toggle-icon" onclick="toggleMobileMenu(this)">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+      <ul class="nav-menu">
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a>Program</a>
+        </li>
+        <li>
+          <a>Join</a>
+        </li>
+        <li>
+          <a>Sponsor</a>
+        </li>
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
```

Comment:
- [optional] It occurred to me that you are repeating this lines of code (same as lines 49 -66), kindly remove it to make your codebase cleaner. Thanks. 👍